Resources/Prototypes/_CD/Roles/Jobs/Civilian/private_investigator.yml

```diff
@@ -22,3 +22,6 @@
     - ForensicPad
     - ForensicPad
     - LogProbeCartridge
+    - TapeRecorder # CD Change
+    - CassetteTape # CD Change
+    - CassetteTape # CD Change
```

Comment:
I think it would be better to replace the three of these with a tape recorder box.

Reply:
Also this does not need CD change comments. This is namespaced.